CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ endif()
 
 set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/Modules ${PROJECT_SOURCE_DIR}/cmake/)
 set(CMAKE_POSITION_INDEPENDENT_CODE 1)
-set(BISON_FLAGS "-Wno-other -Werror=conflicts-sr -Werror=conflicts-rr")
+set(BISON_FLAGS "-Wno-other -Werror=conflicts-sr -Werror=conflicts-rr -Wcounterexamples")
```

Comment:
Just double checked that this flag was introduced in [bison v3.7.5](https://github.com/akimd/bison/blob/25b3d0e1a3f97a33615099e4b211f3953990c203/NEWS#L175), and the `configure.ac` requires at least v3.7.6 :100:.

Reply:
@OverOrion You now have merge/write privileges.